src/manifest.json

```diff
@@ -93,5 +93,5 @@
   "web_accessible_resources": [
     "skin/*",
     "icons/*"
-  ]
+  ],
```

Comment:
The trailing comma makes it invalid JSON (JSON is more constrained than plain JavaScript). Chrome doesn't let you load the extension; all the tests fail because of this.